Library/Homebrew/utils/ruby.sh

```diff
@@ -6,11 +10,19 @@ setup-ruby-path() {
   # When bumping check if HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH (in brew.sh)
   # also needs to be changed.
   local required_ruby_version="2.6"
+  local old_ruby_path
+  local old_ruby_usable
 
   vendor_dir="$HOMEBREW_LIBRARY/Homebrew/vendor"
   vendor_ruby_current_version="$vendor_dir/portable-ruby/current"
   vendor_ruby_path="$vendor_ruby_current_version/bin/ruby"
 
+  if [[ -n $HOMEBREW_RUBY_PATH ]]
+  then
+    old_ruby_path="$HOMEBREW_RUBY_PATH"
+    old_ruby_usable=$(test-ruby "$HOMEBREW_RUBY_PATH")
+  fi
+
   if [[ -z "$HOMEBREW_DEVELOPER" ]]
   then
     unset HOMEBREW_RUBY_PATH
```

Comment:
How about just unsetting this unconditionally? It would allow simplifying the logic above and below

Reply:
Yes, It should be OK to do it now.